source/manual/deploying.html.md

```diff
@@ -5,107 +5,64 @@ parent: "/manual.html"
-[read RFC 70](https://github.com/alphagov/govuk-rfcs/blob/master/rfc-070-path-towards-continuous-deployment-cd.md).
+## 1. Test on integration
 
-### Deployment
+Code that is merged to `master` is automatically deployed to integration. You
+should verify that your changes work there.
 
-1. Check with anyone whose changes you will release during your deploy (check the release app)
-1. Check `#govuk-deploy` recent history and the channel topic
-1. Announce your deployment if it’s potentially problematic
-1. Deploy to staging
-1. Check Smokey passes; remember most apps take a couple of minutes to restart
-1. Check the new functionality works as you would expect
-1. Deploy to production
-1. Check Smokey passes in production; once again, apps take a couple of minutes
-    to restart
-1. Check the new functionality works in production
-1. Take a look at any alerts and metrics, just to check you haven't broken
-   something
-1. Stay around for a while just in case something goes wrong
+## 2. Check the context
 
-An alert for the start and end of your deployment will appear in the channel.
-Jenkins will still enforce sequential deployments per environment across all
-applications, so you may end up in a queue.
+The golden rule is that you only deploy what you can support.
 
-### Holding deployment of applications
+- This means that deploys generally take place between 10am and 4pm, the core hours when most people are in the office.
+- Check `#govuk-deploy` recent history and the channel topic. In some circumstances we may institute short deployment freezes, they will be announced in that channel.
+- Announce your deployment in `#2ndline` if it’s potentially problematic.
 
-If you need to hold deployments of applications, you must add a note to the
-release app. Do this by clicking the edit button in the Notes column in the
-entry for that application.
+## 3. Check your changes
 
-### 2nd line support
+Go to the [Release application](https://release.publishing.service.gov.uk) and find the application you want to deploy. Then select the release tag you want to deploy.
 
-If you need support from 2nd line during your deploy, contact them in advance
-and agree a time.
+- Check what you will be deploying. If there's other people's code to deploy, ask them whether they're okay for the changes to go out.
+- Check if there's a deploy note for the application, to see if there are any special instructions or reasons not to deploy.
 
-### Security patches
+## 4. Deploy to staging
 
-If you are responding to a security incident, follow the steps to
-[deploy fixes for a security vulnerability](deploy-fixes-for-a-security-vulnerability.html).
+From the [Release application](https://release.publishing.service.gov.uk), click the deploy buttons. If you have production access, this will bring you to the Jenkins job to deploy your change.
 
-### Rollback
+Once deployed to staging, let the application run there for 30 minutes. You will need to:
```

Comment:
This is new (apart from Static).  I suggest we communicate this a little more widely than just in the docs.  Maybe worth a recap at tech fortnightly?

I like the sentiment, particularly when catching up with the dependabot deploys.  I might get a bit impatient if it's just my code though.

Reply:
How would you phrase the current practice? Only wait until smokey is finished?

Reply:
I understood it as being the same sorts of instructions, only without the time constraint.

Reply:
So you still need to make sure that your changes actually work etc...

Reply:
I've removed it for now, also because 30 minutes doesn't do much for admin applications - it's only relevant for frontend apps that benefit from traffic replay.